### ISSUE TITLE
Short to SMALLINT

### ISSUE THREAD

**Issue by gorros** [databricks/spark-redshift#401]
If I understand correctly the mapping of Spark data types to Redshift data types, [here](https://github.com/databricks/spark-redshift/blob/master/src/main/scala/com/databricks/spark/redshift/RedshiftJDBCWrapper.scala#L250) we should have `case ShortType => "SMALLINT"`

**Comment by SteveByerly** [comment]
yes, i have been running into this issue and finally figured out it was mapping my `smallint` to `short` and not passing that through to redshift
